### ISSUE TITLE
Accesibility on Scaffolding Create Template

### ISSUE THREAD

**Issue by gtzefrain** [backstage/backstage#5278]
Hello! I'm part of Expedia Group, and noticed that when using a screen reader on the Scaffolder Create Template pages the descriptions on the input fields are inaccessible.
## Expected Behavior
When focusing an input , the screen reader should be able to access the description that's located on the bottom of the input. This will help the user know how to properly fill the field.

## Current Behavior
When focusing an input, the screen reader only reads the label of the input, and indicates if the fields is required.
However, there's no way of accessing using TAB button to the description, and further, is located underneath the fields.
![image](https://user-images.githubusercontent.com/7888169/114236802-c4e4f880-9947-11eb-875c-039541f0e540.png)

## Possible Solution
Add a aria-described by to the inputs, relating the input with the description fields.
https://a11y-demos-web.us-west-2.test.eginternaltools.expedia.com/pages/foundations/aria#aria-describedby
Currently, the span tag which holds the description has no ID however.
I also tried to edit the code on `MultiStepJsonForm.tsx `but this seems to be managed by the `rjsf` package.
If this is isn't covered by the scope of Backstage, let me know.

Make the field accesible via tab key, preferably before the user fills the fields

## Steps to Reproduce

For MAC 

1. Go to the **Create +** page
2. Select any template
3. Click CMD+F5 to enter Voice Over
4. Use TAB key until you reach an input field.

## Context
In Expedia group we have an initiative which aims for Backstage to have better accessibility. Some other teammates, and myself, have created PR changes for this reason.
However, I hit the wall with this one is particular, any help is appreciated.

## Your Environment

- NodeJS Version (v12): v14.16.0
- Operating System and Version (e.g. Ubuntu 14.04): Mac Catalina 10.15.7
- Browser Information: Safari, Chrome

**Comment by benjdlambert** [comment]
Looks like there's an issue on the upstream dependency here which seems it might need to be implemented there: https://github.com/rjsf-team/react-jsonschema-form/issues/959
